app/controllers/container_node_controller.rb

```diff
@@ -7,6 +7,10 @@ class ContainerNodeController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  def show_list
+    process_show_list(:where_clause => 'container_nodes.deleted_on IS NULL')
```

Comment:
Yeah, I saw this change in the other controllers, but it actually doesn't work - archived records are visible in the UI (see https://github.com/ManageIQ/manageiq-ui-classic/issues/1526).